recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -5,6 +5,7 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda
 SECTION = "base"
 
 uixmldir = "${datadir}/nisysapi/uixml"
+settingsdatadir = "${datadir}/sysconfig-settings/systemsettings"
```

Comment:
MINOR: could we use `${PN}` here instead of the `sysconfig-settings` static string?